tsl/src/continuous_aggs/create.c

```diff
@@ -134,8 +137,12 @@ typedef struct MatTableColumnInfo
 	List *matcollist;		 /* column defns for materialization tbl*/
 	List *partial_seltlist;  /* tlist entries for populating the materialization table columns */
 	List *partial_grouplist; /* group clauses used for populating the materialization table */
-	int matpartcolno;		 /*index of partitioning column in matcollist */
-	char *matpartcolname;	/*name of the partition column */
+	List *mat_groupcolname_list; /* names of columns that are populated by the group-by clause
```

Comment:
So, currently it looks like we do it over the mat_groupcolname_list, which is all the group by columns, even if they don't actually end up in the materialization, do we only want to create the indexes if they also appear in the final view? (though this seems like a bit of an edge case)

Reply:
defer this for later.